performance-test/src/puppeteer-test.ts

```diff
@@ -25,19 +46,27 @@ function defer() {
   return promise
 }
 
-function consoleDoneMessage(page: puppeteer.Page) {
-  return new Promise<void>((resolve, reject) => {
+function consoleDoneMessage(
+  page: puppeteer.Page,
+  expectedConsoleMessage: string,
+  errorMessage?: string,
+) {
```

Comment:
I know it's not new code, but could you put a return value on the top level functions to bring it in line with the editor coding style?